src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,47 @@
-package core.basesyntax;
-
-public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
-    }
-}
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class SalaryInfo {
+    private static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final int WORKING_HOURS_INDEX = 2;
+    private static final int RATE_PER_HOURS_INDEX = 3;
+    private static final int EMPLOYEE_NAME_INDEX = 1;
+    private static final int ACTUAL_DATE = 0;
+
+    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
+        StringBuilder stringBuilder = new StringBuilder();

```

Comment:
Let's name your variables more informative, same in all places
```suggestion
        StringBuilder report = new StringBuilder();
```